packages/test/test-end-to-end-tests/src/test/messageSize.spec.ts

```diff
@@ -296,332 +296,356 @@ describeNoCompat("Message size", (getTestObjectProvider) => {
+									`key${i}`,
+									config.payloadGenerator(config.messageSize),
+								);
 							}
-						};
 
-						container.deltaManager.outbound.on("op", handler);
-					});
-				};
-
-				it("Reconnects while sending chunks", async function () {
-					// This is not supported by the local server. See ADO:2690
-					if (provider.driver.type === "local") {
-						this.skip();
-					}
-
-					await setupContainers(chunkingBatchesConfig);
-					// Force the container to reconnect after sending 2 chunked ops,
-					// each in their own batch
-					const secondConnection = reconnectAfterBatchSending(
-						localContainer,
-						(batch) =>
-							batch.length === 1 &&
-							JSON.parse(batch[0].contents)?.type === ContainerMessageType.ChunkedOp,
-						2,
-					);
-
-					await sendAndAssertSynchronization(secondConnection);
-				}).timeout(chunkingBatchesTimeoutMs);
-
-				it("Reconnects while sending compressed batch", async function () {
-					// This is not supported by the local server. See ADO:2690
-					// This test is flaky on tinylicious. See ADO:2964
-					if (
-						provider.driver.type === "local" ||
-						provider.driver.type === "tinylicious"
-					) {
-						this.skip();
-					}
-
-					await setupContainers(chunkingBatchesConfig);
-					// Force the container to reconnect after sending the compressed batch
-					const secondConnection = reconnectAfterBatchSending(
-						localContainer,
-						(batch) =>
-							batch.length > 1 &&
-							batch.slice(1).every((x) => x.contents === undefined),
-						1,
-					);
-
-					await sendAndAssertSynchronization(secondConnection);
-				}).timeout(chunkingBatchesTimeoutMs);
-			});
+							await provider.ensureSynchronized();
+							assertPayloadSize(config.expectedSize);
+							assert.ok(
+								!enableGroupedBatching ||
+									// In case of chunking, we will have more independent messages (chunks) on the wire than in the original batch
+									config.payloadGenerator === generateRandomStringOfSize ||
+									totalOps === 1,
+							);
+						},
+					).timeout(chunkingBatchesTimeoutMs);
+				}));
 		});
 	});
 
-	describe("Payload size on the wire", () => {
-		let totalPayloadSizeInBytes = 0;
-
-		const assertPayloadSize = (totalMessageSizeInBytes: number): void => {
-			// Expecting the message size on the wire should have
-			// at most 35% extra from stringification and envelope overhead.
-			// If any of the tests fail, this value can be increased only if
-			// the payload size increase is intentional.
-			const overheadRatio = 1.35;
-			assert.ok(
-				totalPayloadSizeInBytes < overheadRatio * totalMessageSizeInBytes,
-				`Message size on the wire, ${totalPayloadSizeInBytes} is larger than expected ${
-					overheadRatio * totalMessageSizeInBytes
-				}, after sending ${totalMessageSizeInBytes} bytes`,
-			);
+	describe("Resiliency", () => {
+		const messageSize = 5 * 1024 * 1024;
+		const messagesInBatch = 3;
+		const config: ITestContainerConfig = {
```

Comment:
Is the grouped batching config not needed in these tests?

Reply:
Not really. These tests rely on the batch having multiple ops in the batch.